themes/default/layouts/page/reinvent.html

```diff
@@ -74,7 +77,7 @@ <h5>Pulumi Templates</h5>
                             </p>
                         </div>
                         <div class="card-cta-btn">
-                            <a href="{{ relref . "/challenge" }}" target="_blank" rel="noopener noreferrer" class="btn-primary">Learn More</a>
+                            <a href="{{ relref . "/templates" }}" target="_blank" rel="noopener noreferrer" class="btn-primary">Learn More</a>
```

Comment:
```suggestion
                            <a href="/templates/" target="_blank" rel="noopener noreferrer" class="btn-primary">Learn More</a>
```

We don't use relref anymore so you can just the path on these now.